cfg/admin.txt

```diff
@@ -1,2 +1,23 @@
-jmwturner role=admin
-deitylink role=admin
+unid role=admin
+abi79 role=admin
+arcalane role=admin
+asanadas role=admin
+bobbehluvspropane role=admin
```

Comment:
Okay, this file specifically is used for access to the Reboot/Profile buttons through DM.

I dunno why Deity committed it in the first place, and it's not on .gitignore